### PR TITLE
Popup cards

### PR DESCRIPTION
**Pull request by aamir-asaram**
### Implemented Changes:

- [x] Added JavaScript code that allows project cards in the "My Recent Works" section to be created dynamically.
- [x] Added popup cards when the "See Project" button is clicked. 
- [x] Added popup card layouts for both mobile and desktop versions that switch when the window is resized.
- [x] Added functionality allowing dynamic creation and destruction of popup cards.
- [x] Added button allowing closing of popup cards  
- [x] Added semi-opaque background so that existing web page content is still slightly visible for aesthetic purposes.

### Potential future changes:
- [ ] Add functionality allowing the user to click anywhere outside the card to close/dismiss the card.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -372,6 +368,153 @@ section {
...
+}
+
+.popupTitleDesk {
+  text-align: left;
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  margin-left: 19px;
+  margin-top: 16px;
+  width: 64%;
+}
```
Comment (mukhammadrizooff) [inline styles.css]: - [x] [REQUIRED] Good job so far! Please change the `heading` color to correct one => `color: #172B4D;` 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @aamir-asaram, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -372,6 +369,155 @@ section {
...
+  height: 95vh;
+  width: 86%;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```
Comment (Qoosim) [inline styles.css:395]: - [OPTIONAL] Both the see live and see source buttons do not show on the mobile popup. Kindly check them out and fix.

### Your Design
popup-mobile

### Figma Design
temp-popup-mobile

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**